src/java/org/apache/cassandra/io/util/DataInputPlus.java

```diff
@@ -30,11 +30,31 @@
 @Shared(scope = SIMULATION)
 public interface DataInputPlus extends DataInput
 {
+    /**
+     * Read a 64-bit integer back. This assumes it was originally written using writeVInt
+     * and uses zigzag encoding.
+     * @return
+     * @throws IOException
```

Comment:
nit: remove `@throws IOException` unless it tells us more than the method signature already does

Reply:
Removed it.